installinstallmacos.py

```diff
@@ -269,6 +342,30 @@ def parse_dist(filename):
     return dist_info
 
 
+def get_board_ids(filename):
+    '''Parses a softwareupdate dist file, returning a list of supported
+    Board IDs'''
+    supported_board_ids = ""
+    with open(filename) as search:
+        for line in search:
+            line = line.rstrip()  # remove '\n' at end of line
+            if 'boardIds' in line:
+                supported_board_ids = line.split(" ")[-1][:-1]
+                return supported_board_ids
+
+
+def get_unsupported_models(filename):
+    '''Parses a softwareupdate dist file, returning a list of non-supported
+    ModelIdentifiers'''
+    unsupported_models = ""
+    with open(filename) as search:
```

Comment:
Same comment about XML parsing.

Reply:
Same problem as above.

Reply:
So same comment as above. If 'nonSupportedModels' is never found, the function will return None

What happens if nonSupportedModels are broken over several lines?